_CoqProject

```diff
@@ -1,75 +1,72 @@
--R theories/Core FreeSpec.Core
--R theories/Exec FreeSpec.Exec
--R theories/Stdlib FreeSpec.Stdlib
--R examples FreeSpec.Examples
+-Q _build/default/theories/Core FreeSpec.Core
```

Comment:
_CoqProject is a generated file, right?

Reply:
Unfortunately, no :\ I am not aware of any tools that do that. I need to have one to generate the documentation, since atm `dune` does not support that. But the current setting feels very fragile, to the point where the _order_ of the file in `_CoqProject` matters.

Please, if you have any suggestion to improve the situation, I am all hears!